src/main/java/racingcar/domain/CarPosition.java

```diff
@@ -16,14 +18,23 @@ public CarPosition increasePosition() {
         return new CarPosition(this.position + STEP);
     }
 
-    public boolean isEqaul(int position) {
-        return this.position == position;
-    }
-
-    public int maxPosition(int maxPositioin) {
-        if (this.position > maxPositioin) {
-            return this.position;
+    public CarPosition maxPosition(CarPosition maxPositioin) {
+        if (this.position > maxPositioin.getPosition()) {
```

Comment:
```suggestion
        if (this.position > maxPositioin.position) {
```
개인적으로 동일한 클래스에서는 getter를 사용하지 않고 직접 접근하는편입니다 😄